sdk/extensions/wcf/Microsoft.WCF.Azure.StorageQueues/tests/Microsoft.WCF.Azure.StorageQueues.Tests.csproj

```diff
@@ -0,0 +1,48 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  
+  <PropertyGroup>
+    <RequiredTargetFrameworks>net6.0;net7.0</RequiredTargetFrameworks>
```

Comment:
This will need to run on all targets that our test infrastructure uses.   If there are specific tests/packages that won't run on some, please use conditional package references and hide the tests with an #ifdef.

Reply:
We've fixed up the tests to use all platforms you require and hopefully done it in a way which will continue to work as new .net versions are released with no changes.